conductor/blueprints/authorization/controllers.py

```diff
@@ -69,21 +72,9 @@ def __call__(self):
         return PUBLIC_KEY
 
 
-class Sample:
```

Comment:
@akariv as you already know, we have to refactor the following across this library ( cc @roll this is important ):

* **Always** subclass `object` for classes, otherwise we actually have different class behaviour over Python 2 and 3! (`super`, etc.)
* **Never** use `__call__`. I can' think of any use case we need it, and it is idiomatic to use `__init__` in Python, in the places that `__call__` is used here